ui/trailingcolorlabel.cpp

```diff
@@ -0,0 +1,64 @@
+#include "trailingcolorlabel.h"
+
+#include <QPainter>
+#include <QPaintEvent>
+#include <QStyle>
+#include <QDebug>
+
+namespace GammaRay {
+
+TrailingColorLabel::TrailingColorLabel(QWidget *parent) : QLabel(parent)
+{
+    setAttribute(Qt::WA_TransparentForMouseEvents);
+    setMinimumSize(QSize(145, 30));
+    m_pickedColor = qRgba(0,0,0,0);
+    setHidden(true);
+}
+
+void TrailingColorLabel::paintEvent(QPaintEvent *event)
+{
+    Q_UNUSED(event);
+    QPainter *p = new QPainter(this);
+    p->setPen(QPen(Qt::NoPen));
+    const auto palette = this->style()->standardPalette();
+    p->setBrush(palette.base());
+    p->drawRect(0,0,145,30);
```

Comment:
This, and the numbers below seem to assume a specific font size, which is likely not the same everywhere. Probably needs to depend on fontMetrics() instead.